Add GCE Eco-Devices integration

Aohzan opened the pull request:
## Proposed change

Add documentation for a new integration for GCE Eco-Devices: https://www.gce-electronics.com/fr/carte-relais-ethernet-module-rail-din/409-teleinformation-ethernet-ecodevices.html
From custom component https://github.com/Aohzan/ecodevices

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase:  https://github.com/home-assistant/core/pull/45499
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/2203
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

Aohzan committed:
add ecodevices integration

homeassistant commented:
Hi @shawnoster, @kongo09, @matt-gnu, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#16295).

Once you do that we will be able to review and accept this pull request.

Thanks!

Aohzan commented:
> It seems that this PR is targeted against an incorrect branch. Documentation updates which apply to our current stable release should target the `current` branch. Please change the target branch of this PR to `current` and rebase if needed. If this is documentation for a new feature, please add a link to that PR in your description.
> (message by DocsTargetBranch)

Hello, I switch the branch to current, but I did on the next branch because : 
> Added documentation for a new integration I'm adding to Home Assistant (next branch)

The link is already on the first post : 
> Link to parent pull request in the codebase: home-assistant/core#45499

frenck commented on source/_integrations/ecodevices.markdown:
````diff
@@ -0,0 +1,124 @@
+```yaml
+# Example configuration.yaml entry
+sensor:
+  - platform: ecodevices
+    host: "192.168.1.239"
+    t1_enabled: True
+    t1_name: Compteur Linky
+    c1_enabled: True
+    c1_name: Compteur panneau solaire
+    c1_icon: mdi:solar-panel
+    c1_unit_of_measurement: Wh
+    c1_device_class: power
````
New integrations, that communicate with a device or service, should not provide an option for YAML configuration.

Please remove this functionality, as it is only intended for transitional/migration purposes of existing integrations.

Aohzan replied:
Ok, I will update it to delete the YAML part and provide only configuration from GUI.

For "complexe" integration (like https://github.com/Aohzan/ipx800#example, that I will subit soon too) which need a lot of parameter. The YAML isn't possible too ?

frenck replied:
Exceptions are possible, for example, MQTT/KNX, which define a complex protocol/specification that isn't possible with the UI at this point.

However, in this PR; that is not the case, as a config flow is provided already.

Aohzan replied:
yes of course, so I have to think about the ipx800v4 integration

Aohzan committed:
remove yaml configuration from ecodevices integration

Aohzan committed:
update ha_release

Aohzan commented:
done @frenck :)

klaasnicolaas commented on source/_integrations/ecodevices.markdown:
```diff
@@ -0,0 +1,18 @@
+
+The `ecodevices` integration allows you to get information from [GCE Eco-Devices](http://gce-electronics.com/fr/carte-relais-ethernet-module-rail-din/409-teleinformation-ethernet-ecodevices.html).
+
+## Configuration
+
+To add ecodevices to your installation, go to Configuration >> Integrations in the UI, click the button with + sign and from the list of integrations select GCE Eco-Devices.
```
```suggestion
{% include integrations/config_flow.md %}
```

Aohzan replied:
thanks

klaasnicolaas reviewed (dismissed):
✅ Approved. Can be merged as soon as the parent PR gets merged 👍
⚠️ The ha_release might need an update before merging this one in.

raman325 committed:
Document zwave_js.reset_meter service (#18587)

* Document zwave_js.meter_reset service

* revert additional change and put in separate PR

* Updates based on PR feedback

* Tweak

* tweaks based on upstream feedback

Co-authored-by: Franck Nijhof <frenck@frenck.nl>

balloob committed:
Add energy statistics more info (#18657)

balloob committed:
Add energy statistics more info (#18657)

balloob committed:
Update the in-page anchor (#18659)

balloob committed:
Update the in-page anchor (#18659)

frenck committed:
Revert "Documentation for new Automate integration" (#18656)

This reverts commit 4503957bea8a6949bd6c75910daf6e3018a71a35.

frenck committed:
Revert "Documentation for new Automate integration" (#18656)

This reverts commit 4503957bea8a6949bd6c75910daf6e3018a71a35.

frenck committed:
2021.8.0 beta notes tweaks

ludeeus committed:
Add default value for active installations (#18666)

eroldan committed:
Update tplink.markdown (#18664)

Corrected wrong template name example.

peyanski committed:
Update hikvision.markdown (#18667)

The stated Annke NVR is also confirmed working. As reference check this article - https://peyanski.com/annke-c800-annke-nvr-home-assistant-integration/

dependabot[bot] committed:
Bump ruby/setup-ruby from 1.78.0 to 1.79.0 (#18688)

Bumps [ruby/setup-ruby](https://github.com/ruby/setup-ruby) from 1.78.0 to 1.79.0.
- [Release notes](https://github.com/ruby/setup-ruby/releases)
- [Commits](https://github.com/ruby/setup-ruby/compare/v1.78.0...v1.79.0)

---
updated-dependencies:
- dependency-name: ruby/setup-ruby
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

koying committed:
FIX: It's "trace:", not "traces:" (#18683)

Not actually sure if it's a doc or code bug, but "traces:" is refused while "trace:" works

emontnemery committed:
Clarify light `white` turn_on parameter has no corresponding state attribute (#18674)

emontnemery committed:
Update scene examples (#18675)

pdcemulator committed:
edl21: Add DZG DWS76 as tested smart meter (#18655)

nickovs committed:
Document change to Lutron button events for un-named buttons (#18650)

nickovs committed:
Document change to Lutron button events for un-named buttons (#18650)

janiversen committed:
Allow combinations write_coil/read_coils and write_coils/read_coil for modbus switch (#18691)

janiversen committed:
Allow combinations write_coil/read_coils and write_coils/read_coil for modbus switch (#18691)

ludeeus committed:
Add energy analytics (#18693)

ludeeus committed:
Add energy analytics (#18693)

bramkragten committed:
New Lovelace element 2021.8 (#18695)

bramkragten committed:
New Lovelace element 2021.8 (#18695)

bieniu committed:
Add support for MJJSQ humidifiers for Xiaomi MIIO integration (#18678)

bramkragten committed:
Masonry typo (#18699)

bramkragten committed:
Masonry typo (#18699)

ludeeus committed:
Update about section (#18702)

Co-authored-by: Joakim Sørensen <joasoe@gmail.com>
Co-authored-by: Paulus Schoutsen <balloob@gmail.com>

dependabot[bot] committed:
Bump nokogiri from 1.11.7 to 1.12.0 (#18710)

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

ludeeus committed:
Add Home Energy Management to features (#18711)

* Add Home Energy Management to features

* Update source/_includes/custom/features.html

ludeeus committed:
Add Home Energy Management to features (#18711)

* Add Home Energy Management to features

* Update source/_includes/custom/features.html

frenck committed:
Merge branch 'current' into next

AaronDavidSchneider committed:
extend fritz doc (#18682)

* extend fritz doc

* fix FRITZ!Box name

* Update source/_integrations/fritz.markdown

Co-authored-by: Simone Chemelli <simone.chemelli@gmail.com>

* Update source/_integrations/fritz.markdown

Co-authored-by: Franck Nijhof <frenck@frenck.nl>

* Update source/_integrations/fritz.markdown

Co-authored-by: Franck Nijhof <frenck@frenck.nl>

* suggestions

Co-authored-by: Simone Chemelli <simone.chemelli@gmail.com>
Co-authored-by: Franck Nijhof <frenck@frenck.nl>

AaronDavidSchneider committed:
extend fritz doc (#18682)

* extend fritz doc

* fix FRITZ!Box name

* Update source/_integrations/fritz.markdown

Co-authored-by: Simone Chemelli <simone.chemelli@gmail.com>

* Update source/_integrations/fritz.markdown

Co-authored-by: Franck Nijhof <frenck@frenck.nl>

* Update source/_integrations/fritz.markdown

Co-authored-by: Franck Nijhof <frenck@frenck.nl>

* suggestions

Co-authored-by: Simone Chemelli <simone.chemelli@gmail.com>
Co-authored-by: Franck Nijhof <frenck@frenck.nl>

balloob committed:
Document date selector (#18709)

balloob committed:
Document date selector (#18709)

fabaff committed:
Add note that only TTNv2 is supported (#18703)

balloob committed:
Energy updates (#18708)

* Energy updates

* Tweak

Co-authored-by: Franck Nijhof <git@frenck.dev>

balloob committed:
Energy updates (#18708)

* Energy updates

* Tweak

Co-authored-by: Franck Nijhof <git@frenck.dev>

bramkragten committed:
Correct statistics text (#18701)

`/integrations/statistics/` is something else...

bramkragten committed:
Correct statistics text (#18701)

`/integrations/statistics/` is something else...

TastyPi committed:
Universal media player toggle (#17558)

This describes the behavior being added in home-assistant/core#49395

timmo001 committed:
System Bridge - Data from WebSocket - Docs Update (#18603)

mib1185 committed:
Move power and energy attributes to sensors (#18644)

mib1185 committed:
Move power and energy attributes to sensors (#18644)

pedrolamas committed:
Adds "limits" configuration to Lovelace sensor docs (#18694)

frenck committed:
Merge branch 'current' into next

frenck committed:
Merge branch 'current' into rc

slovdahl committed:
More numeric_state attribute/value_template docs (#18696)

This PR adds a bit more documentation for the `attribute` and `value_template` properties for the `numeric_state` trigger.

bramkragten committed:
Lovelace typos (#18712)

bramkragten committed:
Lovelace typos (#18712)

dgomes committed:
Fixes to Energy related documentation (#18700)

* Update faq.markdown

* fix units

* Apply suggestions from code review

Co-authored-by: Franck Nijhof <frenck@frenck.dev>

* rewriting

Co-authored-by: Franck Nijhof <frenck@frenck.dev>

dgomes committed:
Fixes to Energy related documentation (#18700)

* Update faq.markdown

* fix units

* Apply suggestions from code review

Co-authored-by: Franck Nijhof <frenck@frenck.dev>

* rewriting

Co-authored-by: Franck Nijhof <frenck@frenck.dev>

bramkragten committed:
Update feedback card styling (#18713)

frenck committed:
Merge branch 'current' into next

frenck committed:
Merge branch 'current' into rc

frenck committed:
2021.8.0 beta notes updates

ludeeus committed:
Add energy landingpage (#18689)

Co-authored-by: Franck Nijhof <git@frenck.dev>
Co-authored-by: Paulus Schoutsen <balloob@gmail.com>

ludeeus committed:
Add energy landingpage (#18689)

Co-authored-by: Franck Nijhof <git@frenck.dev>
Co-authored-by: Paulus Schoutsen <balloob@gmail.com>

frenck committed:
2021.8.0 beta notes language tweaks

frenck committed:
2021.8.0 beta notes fix broken links

balloob committed:
Tweaks landing page

adamjernst committed:
Add stop service to Rachio (#18668)

balloob committed:
Document link_dashboard

balloob committed:
Add title shadow and mobile padding

balloob committed:
Add frontpage pictures for landing pages

ludeeus committed:
Use frontpage images (#18718)

balloob committed:
Make energy hero smaller

balloob committed:
Add badges

balloob committed:
Add link to landingpage for people with HA

balloob committed:
Reduce font a bit

dependabot[bot] committed:
Bump nokogiri from 1.12.0 to 1.12.1 (#18719)

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
Bump remark-cli from 9.0.0 to 10.0.0 (#18720)

Bumps [remark-cli](https://github.com/remarkjs/remark) from 9.0.0 to 10.0.0.
- [Release notes](https://github.com/remarkjs/remark/releases)
- [Changelog](https://github.com/remarkjs/remark/blob/main/changelog.md)
- [Commits](https://github.com/remarkjs/remark/compare/remark-cli@9.0.0...10.0.0)

---
updated-dependencies:
- dependency-name: remark-cli
  dependency-type: direct:development
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
Bump actions/setup-node from 2.3.0 to 2.3.1 (#18722)

Bumps [actions/setup-node](https://github.com/actions/setup-node) from 2.3.0 to 2.3.1.
- [Release notes](https://github.com/actions/setup-node/releases)
- [Commits](https://github.com/actions/setup-node/compare/v2.3.0...v2.3.1)

---
updated-dependencies:
- dependency-name: actions/setup-node
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
Bump remark-stringify from 9.0.1 to 10.0.0 (#18721)

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

frenck committed:
Merge branch 'current' into next

frenck committed:
Merge branch 'current' into rc

frenck committed:
2021.8.0 Update release for final

frenck committed:
Sync codebase with docs for 2021.8

frenck committed:
Add blog energy management (#18726)

Co-authored-by: Joakim Sørensen <joasoe@gmail.com>
Co-authored-by: Paulus Schoutsen <balloob@gmail.com>

frenck committed:
Fix broken link in Energy Management blog

frenck committed:
Redirect energy integration to energy docs (#18727)

wschoot committed:
typo (#18729)

Aohzan committed:
Merge branch 'home-assistant:current' into current

hmmbob committed:
Add energy to `default_config` list (#18732)

balloob committed:
2021.8.1

dependabot[bot] committed:
Bump actions/setup-node from 2.3.1 to 2.3.2 (#18765)

Bumps [actions/setup-node](https://github.com/actions/setup-node) from 2.3.1 to 2.3.2.
- [Release notes](https://github.com/actions/setup-node/releases)
- [Commits](https://github.com/actions/setup-node/compare/v2.3.1...v2.3.2)

---
updated-dependencies:
- dependency-name: actions/setup-node
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
Bump ruby/setup-ruby from 1.79.0 to 1.80.0 (#18764)

Bumps [ruby/setup-ruby](https://github.com/ruby/setup-ruby) from 1.79.0 to 1.80.0.
- [Release notes](https://github.com/ruby/setup-ruby/releases)
- [Commits](https://github.com/ruby/setup-ruby/compare/v1.79.0...v1.80.0)

---
updated-dependencies:
- dependency-name: ruby/setup-ruby
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
Bump nokogiri from 1.12.1 to 1.12.2 (#18763)

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

pedrolamas committed:
Fixes the currency configuration to use ISO 4217 (#18734)

epenet committed:
Add new component renault documentation (#14383)

Co-authored-by: Milan Meulemans <milan.meulemans@live.be>
Co-authored-by: Franck Nijhof <git@frenck.dev>

starkillerOG committed:
add lumi.gateway.aqhm01 support (#18724)

ludeeus committed:
Adjust documentation for config flow (#18728)

snakuzzo committed:
Update media_player.markdown (#18768)

Add device_class: receiver according to dev docs https://developers.home-assistant.io/docs/core/entity/media-player/

kukulich committed:
Typo in statistics-graph options (#18743)

pdcemulator committed:
Remove comma (#18771)

Ernst79 committed:
Add energy tag to solarlog docs (#18745)

* Add energy tag to solarlog docs

* Change category in stead of platforms

zhulik committed:
Add docs for Tractive (#17939)

* Add docs for Tractive

* Remove quality scale

* Update source/_integrations/tractive.markdown

Co-authored-by: Franck Nijhof <frenck@frenck.nl>

IonutNeagu committed:
Update rate limit (#18715)

pedrolamas committed:
Fixes missing forward slash prefix on url (#18785)

balloob committed:
2021.8.2

tomlut committed:
SMA Integration: Correct energy meter entity descriptions (#18792)

jwillemsen committed:
Updated to correct indent (#18807)

jwillemsen committed:
Update pvoutput.markdown, corrected identation (#18806)

balloob committed:
2021.8.3

jlweston committed:
Fix broken link in Tautulli integration docs (#18810)

* Fix broken link in Tautulli integration docs

* :pencil2: Tweak

Co-authored-by: Klaas Schoute <klaas_schoute@hotmail.com>

pfunkmallone committed:
Change "remove" to "delete" in zwave_js tutorial (#18813)

Just trying to match the actual gui

sotatech committed:
Update generic_hygrostat.markdown (#18844)

Correct use of temperature instead of humidity.

saintmalik committed:
Fix broken url in docs (#18833)

What does this PR do

Fix dead or broken links in this docs page, making navigation easy for others.

pedrolamas committed:
Fixes link in scripts.markdown and schema.markdown (#18826)

nickrout committed:
Small error in grammar (#18797)

"who" denotes a person, "which" a thing.

koying committed:
local_file: typo (#18796)

Wrong service name

tomlut committed:
Pedantic corrections (#18845)

Power and energy are different things.

clach04 committed:
Tag doc typo (#18855)

ludeeus committed:
Add changelog for 2021.8.4 (#18858)

dependabot[bot] committed:
Bump ruby/setup-ruby from 1.80.0 to 1.81.0 (#18857)

Bumps [ruby/setup-ruby](https://github.com/ruby/setup-ruby) from 1.80.0 to 1.81.0.
- [Release notes](https://github.com/ruby/setup-ruby/releases)
- [Commits](https://github.com/ruby/setup-ruby/compare/v1.80.0...v1.81.0)

---
updated-dependencies:
- dependency-name: ruby/setup-ruby
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

BreakingBread0 committed:
Update tesla.markdown (#18750)

* Update tesla.markdown

* Update tesla.markdown

chrisrosset committed:
Remove broken ISS camera snippet (#18814)

CircuitSetup committed:
Update electricity-grid.markdown (#18812)

Add link to ESPHome ATM90E32 energy meter sensor.

Commit:
Update dsmr.markdown (#18811)

Hi! I've archived this repo and @daniel-jong is maintaining a fork of the DIY, so please adjust the url for a more recent and maintained version :)

I doubt that it's still needed with the new [project by Marcel Zuidwijk](https://github.com/zuidwijk/dsmr/)) but for completeness :)

willhayslett committed:
Fixed typo and cleaned up language for Lovelace Card documentation (#18808)

ganey committed:
Update modbus markdown typo (#18805)

Update "binary_ensor2" to "binary_sensor2"

spinydelta committed:
Update forecast_solar.markdown (#18801)

ohadbenita committed:
Switcher KIS Documentation: Add clarification about old firmware versions. (#18818)

* Update switcher_kis.markdown

@thecode FYI.

* Update source/_integrations/switcher_kis.markdown

Co-authored-by: Franck Nijhof <frenck@frenck.nl>

Co-authored-by: Franck Nijhof <frenck@frenck.nl>

dependabot[bot] committed:
Bump remark-frontmatter from 3.0.0 to 4.0.0 (#18791)

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
Bump actions/setup-node from 2.3.2 to 2.4.0 (#18790)

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

Hypfer committed:
Update humidifier.mqtt.markdown (#18776)

rosscullen committed:
Smappee Integration - minor note re mDNS (#18774)

Line added as per discussion with @bsmappee regarding default mDNS and password settings on Smappee device local web portal.

jbouwh committed:
Fix Xiaomi-miio humidifiers - correct modes and document attributes (#18867)

Co-authored-by: Maciej Bieniek <bieniu@users.noreply.github.com>

gagebenne committed:
Clarify login credentials and prerequisites (#18866)

* Clarify login credentials and prerequisites

* Revise to pass textlint

bieniu committed:
Add information about light transition for Shelly integration (#18862)

* Add information about light transition

* Sort list of models

* Remove empty line

camcs1 committed:
Add an example template to extract air quality (#18869)

* Add an example template to extract air quality 

Thought it would be helpful to add a example template showing how to extract air quality and capitalise the results. 

This is my first PR for this project so I apologise if I've made any mistakes

* Use new template format & changed to state_attr()

* Tweaks

* Tweak

Co-authored-by: Franck Nijhof <frenck@frenck.nl>

farmio committed:
Add `state_class` to KNX sensor (#18782)

bieniu committed:
Add `binary_sensor` documentation for Xiaomi Miio integration (#18793)

r-t-s committed:
Add service homeassistant.save_persistent_states (#18697)

balloob committed:
2021.8.5

balloob committed:
2021.8.5 - pt2

thorsten committed:
Update Stiebel Eltron supported units (#18873)

raman325 committed:
Update Climacell rate limit (#18872)

janiversen committed:
Add message_wait_milliseconds for modbus (#18821)

raman325 committed:
Make note about new siren feature (#18856)

bachya committed:
Add ecowitt2mqtt to MQTT Discovery docs (#18879)

balloob committed:
Remove wink from Z-Wave guide (#18880)

Aeotec-ccheng committed:
Update to Supported USB Sticks (#18883)

balloob committed:
2021.8.6

dependabot[bot] committed:
Bump nokogiri from 1.12.2 to 1.12.3 (#18887)

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

janiversen committed:
Deprecate wink. (#18884)

* Deprecate wink.

* Removed extra copy/paste

* Update source/_integrations/wink.markdown

Co-authored-by: Franck Nijhof <git@frenck.dev>

Co-authored-by: Franck Nijhof <git@frenck.dev>

robot256 committed:
Update category of Greeney Monitor per #18741 (#18894)

Add category tag "Energy" to the GreenEye Monitor (GEM) integration, addressing Issue #18741

Ghannes committed:
add iCloud example (#18904)

frenck committed:
Remove broken links from Serial Particulate Matter integration page (#18893)

shanem2004 committed:
Update tplink.markdown include device KP40 support (#18903)

Co-authored-by: Really Good Tech Deals <51071512+iKingRPG@users.noreply.github.com>

raman325 committed:
Add number/siren category and platform to zwave_js page metadata (#18909)

* Add siren category and platform to zwave_js page metadata

* add number platform as well

balloob committed:
2021.8.7

dependabot[bot] committed:
Bump remark-lint-no-shell-dollars from 2.0.2 to 3.0.0 (#18919)

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
Bump remark-lint from 8.0.0 to 9.0.0 (#18918)

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

fmartinou committed:
Add Teleinfo Mqtt and What'sup Docker? to the list of the third-party… (#18923)

jnimmo committed:
Remove intesishome logo after migrating logos to Brands repo (#18930)

AlecRust committed:
Small Nest docs tweaks/fixes (#18931)

cnico committed:
correction on sensor name (#18937)

ludeeus committed:
Remove alternatives from version (#18936)

bieniu committed:
Add information about sensor platform for fan (#18912)

dependabot[bot] committed:
Bump remark-lint-fenced-code-flag from 2.0.1 to 3.0.0 (#18917)

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

hiveai committed:
Fix for Issure #18933: change "install media" to "boot media" and add more detailed descript… (#18934)

chriscn committed:
Fixed minor grammer issue in electricity-grid.markdown (#18956)

raman325 committed:
Add select platform and category to zwave_js docs (#18915)

bdraco committed:
Convert nmap_tracker to be a config flow (#18955)

Jc2k committed:
Add new air quality device classes (#18957)

igrybkov committed:
Change above/below confusion in clarification of example (#18739)

mib1185 committed:
Add device class update (#18733)

air2 committed:
Added ZIV as working with the DSMR integration (#18964)

ehendrix23 committed:
Add documentation for new upnp binary sensor (#18926)

Bert-R committed:
Removed now obsolete max_results (#18830)

My PR home-assistant/core#54231 obsoletes the max_results configuration

raman325 committed:
Add Binary Sensor category for template integration doc (#18976)

dependabot[bot] committed:
Bump remark-lint from 9.0.0 to 9.0.1 (#18974)

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
Bump dessant/lock-threads from 2.1.1 to 2.1.2 (#18973)

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
Bump remark-lint-fenced-code-flag from 3.0.0 to 3.0.1 (#18971)

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
Bump remark-lint-no-shell-dollars from 3.0.0 to 3.0.1 (#18972)

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

bieniu committed:
Add information about `select` platform for Xiaomi Miio fans (#18966)

ehendrix23 committed:
Add light to MyQ (#18928)

bdraco committed:
Clarify which devices have been tested with the august integration (#18970)

peca89 committed:
Add an example for calling the Pi-hole disable service (#18945)

balloob committed:
2021.8.8

dependabot[bot] committed:
Bump listen from 3.6.0 to 3.7.0 (#18993)

Bumps [listen](https://github.com/guard/listen) from 3.6.0 to 3.7.0.
- [Release notes](https://github.com/guard/listen/releases)
- [Commits](https://github.com/guard/listen/compare/v3.6.0...v3.7.0)

---
updated-dependencies:
- dependency-name: listen
  dependency-type: indirect
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

crazyfx1 committed:
Fix confusing typo in vicare (#18994)

frenck committed:
Merge branch 'current' into next

linus-ha committed:
"iframe Panel" better example (#18985)

linus-ha committed:
"iframe Panel" better example (#18985)

emontnemery committed:
Improve DSMR documentation and document support for Swedish meters (#18996)

reedy committed:
Fix formatting of numbers in prefix warning box (#19001)

reedy committed:
Fix formatting of numbers in prefix warning box (#19001)

balloob committed:
Document Zigbee energy profile (#19003)

balloob committed:
Document Zigbee energy profile (#19003)

dcs8 committed:
'Monetary' missing from list of Device Classes (#19005)

dcs8 committed:
'Monetary' missing from list of Device Classes (#19005)

klaasnicolaas committed:
Add docs for P1 Monitor integration (#19000)

bieniu committed:
Add information about `switch` platform for Xiaomi Miio fans (#19012)

elupus committed:
Add fjäråskupan documentation (#18542)

Co-authored-by: Klaas Schoute <klaas_schoute@hotmail.com>
Co-authored-by: Martin Hjelmare <marhje52@gmail.com>

raman325 committed:
Document area_id as an input to zwave_js services (#19016)

lirantal committed:
docs: update required paid plan for using this field (#19017)

lirantal committed:
docs: update required paid plan for using this field (#19017)

bdraco committed:
Document known discoverable zwave_js devices (#19023)

bieniu committed:
Add information about `number` platform for Xiaomi Miio fans (#19025)

bdraco committed:
Add ZWaveMe stick to zwave_js known discoverable devices (#19028)

* Add ZWaveMe stick to zwave_js known discoverable devices

* Update source/_integrations/zwave_js.markdown

janiversen committed:
Lazy error in modbus (#18878)

Co-authored-by: Franck Nijhof <frenck@frenck.nl>

balloob committed:
Update CT clamp docs (#19030)

balloob committed:
Update CT clamp docs (#19030)

bdraco committed:
Document known discoverable zha devices (#19022)

Co-authored-by: Franck Nijhof <git@frenck.dev>

bdraco committed:
Add Usb discovery (#19015)

Co-authored-by: Franck Nijhof <git@frenck.dev>

samsinnamon committed:
Airtouch4 integration (#15717)

Co-authored-by: Franck Nijhof <git@frenck.dev>

Dycelll committed:
You need a Pro subscription to access API (#18802)

Dycelll committed:
You need a Pro subscription to access API (#18802)

tels7ar committed:
Fix broken code link, edit for clarity. (#18882)

Co-authored-by: Franck Nijhof <git@frenck.dev>

tels7ar committed:
Fix broken code link, edit for clarity. (#18882)

Co-authored-by: Franck Nijhof <git@frenck.dev>

bdraco committed:
Update when usb devices are detected (#19032)

frenck committed:
Small tweak to frontend integration page (#19034)

frenck committed:
Small tweak to frontend integration page (#19034)

micha91 committed:
Add documentation for play media in yamaha_musiccast (#18995)

Co-authored-by: Franck Nijhof <frenck@frenck.nl>

raman325 committed:
Add documentation on zwave_js automations (#19008)

Co-authored-by: Franck Nijhof <frenck@frenck.nl>

nielstron committed:
Add notes on the new logger info device for fronius (#18978)

Co-authored-by: Matthias Alphart <farmio@alphart.net>

tkdrob committed:
Add quality scale for goalzero (#18575)

Jc2k committed:
Add volatile organic compounds device class (#19053)

balloob committed:
Add basic battery/gas docs (#19049)

balloob committed:
Eagle-200 docs (#19048)

richid committed:
Update tplink.markdown: include device EP40 support (#19058)

richid committed:
Update tplink.markdown: include device EP40 support (#19058)

Ernst79 committed:
Fix solarlog unit of measurements (#19061)

bieniu committed:
Add information about `sensor` platform for Tractive integration (#18799)

Ernst79 committed:
Remove Wp (watt-peak) from unit of measurements (#19067)

ludeeus committed:
Add blogpost about upcomming supervisor changes (#18493)

Co-authored-by: Franck Nijhof <git@frenck.dev>
Co-authored-by: Stefan Agner <stefan@agner.ch>

ludeeus committed:
Add blogpost about upcomming supervisor changes (#18493)

Co-authored-by: Franck Nijhof <git@frenck.dev>
Co-authored-by: Stefan Agner <stefan@agner.ch>

frenck committed:
Adjust homeassistant services description to clarify their purpose (#19035)

frenck committed:
Adjust homeassistant services description to clarify their purpose (#19035)

frenck committed:
Remove non-existing services from Kostal Plenticore documentation (#19036)

frenck committed:
Remove non-existing services from Kostal Plenticore documentation (#19036)

frenck committed:
Clarify target example in script service call documentation (#19037)

frenck committed:
Clarify target example in script service call documentation (#19037)

frenck committed:
Fix typo in fido documentation (#19038)

frenck committed:
Fix typo in fido documentation (#19038)

frenck committed:
Fixes typo on Derivative page (#19033)

frenck committed:
Fixes typo on Derivative page (#19033)

frenck commented:
There are unrelated commits in this PR. Please git rebase your changes onto the latest `next` branch to clean it up. Thanks 👍

kkr16 committed:
Fixed Hue bulb reset instructions (#18885)

kkr16 committed:
Fixed Hue bulb reset instructions (#18885)

pvizeli committed:
Change description for unhealthy (#19057)

Co-authored-by: Franck Nijhof <git@frenck.dev>

pvizeli committed:
Change description for unhealthy (#19057)

Co-authored-by: Franck Nijhof <git@frenck.dev>

Aohzan committed:
Merge branch 'home-assistant:current' into current

Aohzan committed:
rebase

Aohzan committed:
merge

Aohzan commented:
I close this PR as I messed up with by git...
